site/app/views/admin/LibraryManagerView.php

```diff
@@ -2,12 +2,17 @@
 
```

Comment:
Oh sweet! You guys already made the view! Should be super easy to hook up and display.